clouddriver-azure/src/main/groovy/com/netflix/spinnaker/clouddriver/azure/client/AzureResourceManagerClient.groovy

```diff
@@ -297,8 +307,13 @@ class AzureResourceManagerClient extends AzureBaseClient {
   }
 
   @Canonical
-  private static class ParameterValue {
-    String value
+  private static class ValueParameter {
+    Object value
+  }
+
+  @Canonical
+  static class ReferenceParameter {
```

Comment:
ValueParameter class is private. Should this one be private as well? They should be consistent.